File: app/src/main/java/com/attendance/contract/SplashContract.java
package com.attendance.contract;

import com.attendance.BasePresenter;
import com.attendance.BaseView;

/**
 * Created by peiqin on 3/4/2017.
 */

public interface SplashContract {

    interface View extends BaseView<Presenter> {

        void startLoginActivity();

        void startMainActivity();

    }

    interface Presenter extends BasePresenter {

        void startTimeTask();

    }

}

File: app/src/main/kotlin/com.attendance/contract/SplashContract.kt
package com.attendance.contract

import com.attendance.BasePresenter
import com.attendance.BaseView

/**
 * Created by peiqin on 3/4/2017.
 */

interface SplashContract {

    interface View : BaseView<Presenter> {

        fun startLoginActivity()

        fun startMainActivity()

    }

    interface Presenter : BasePresenter {

        fun startTimeTask()

    }

}
